scripts/embed_chunks.py

```
@@ -0,0 +1,97 @@
+    # process each chunks jsonl file
+    for f in CHUNKS_DIR.glob('*.chunks.jsonl'):
+        sha = f.stem.split('.')[0]
+        texts = []
+        metas = []
+        for line in f.open('r', encoding='utf-8'):
+            try:
+                obj = json.loads(line)
+                texts.append(obj.get('text',''))
+                metas.append({'chunk_id': obj.get('chunk_id'), 'preview': obj.get('preview')})
+            except Exception:
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)